common/server_protocolhandler.cpp

```diff
@@ -373,6 +390,22 @@ void Server_ProtocolHandler::pingClockTimeout()
 
     if (timeRunning - lastDataReceived > server->getMaxPlayerInactivityTime())
         prepareDestroy();
+ 
+    if (QString::fromStdString(userInfo->privlevel()).toLower() == "none")
```

Comment:
Hiding this conditional under a `userMayBeTimedOutFromInactivity` would be good - it would give a place for a comment to live explaining that it only applies to non-{mod/admin/vip} get timed out